packages/twenty-ui/src/components/chip/LinkChip.tsx

```diff
@@ -14,7 +14,8 @@ export type LinkChipProps = Omit<
   'onClick' | 'disabled' | 'clickable'
 > & {
   to: string;
-  onClick?: (event: MouseEvent<HTMLAnchorElement>) => void;
+  onClick?: (event: MouseEvent<HTMLElement>) => void;
```

Comment:
change: Feels a bit wrong to pass the onClick / onMouseDown to LinkAvatarChip and LinkChip, I feel they should call the hook internally and the current onClick is a onNavigate that should be called either way

Reply:
Refactored to do it at a lower level